tools/fast/series/series_sendfilecoindefaults.go

```diff
@@ -0,0 +1,19 @@
+package series
+
+import (
+	"context"
+
+	"github.com/filecoin-project/go-filecoin/address"
+	"github.com/filecoin-project/go-filecoin/tools/fast"
+)
+
+// SendFilecoinDefaults sends the `value` amount of fil from the default wallet
+// address of the `from` node to the `to` node's default wallet.
+func SendFilecoinDefaults(ctx context.Context, from, to *fast.Filecoin, value int) error {
+	var toAddr address.Address
+	if err := to.ConfigGet(ctx, "wallet.defaultAddress", &toAddr); err != nil {
```

Comment:
we should have a command that returns the default address using a plumbing call that is higher level than config because it might not yet be set. maybe file an issue on the plumbing/porcelain epic to have plumbing that returns the default address.

Reply:
https://github.com/filecoin-project/go-filecoin/issues/2049